sharpy/__init__.py

```diff
@@ -96,6 +96,17 @@ def _validate_device(device):
         raise ValueError(f"Invalid device string: {device}")
 
 
+def arange(start, stop=None, step=1, dtype=int64, device="", team=1):
+    if stop is None:
+        stop = start
+        start = 0
```

Comment:
Here you are adding a new way to call arange: `arange(stop)` which is equal to `arange(0, stop)`. This is useful, but this route should be tested too. Please add a test in test/test_create.py. Alternatively you can drop this change from this PR (unrelated to permute_dims).

Reply:
removed